os/fs/procfs/Make.defs

```diff
@@ -66,6 +66,7 @@ endif
 ifeq ($(CONFIG_ARCH_BOARD_SIDK_S5JT200),y)
 CFLAGS+=-I$(TOPDIR)/../apps/include/netutils/wifi
 endif
+CFLAGS+=-I$(TOPDIR)/pm
```

Comment:
If pm is config, it would be better to add it.

Reply:
pm is directory name.